documentation/src/getting_started.md

```diff
@@ -1,19 +1,100 @@
+
+- A computer
+  - OS either:
+    - Windows 10/11
+      - Windows 7/8/8.1 might work, but untested
+    - Linux
+      - Ubuntu 20.04 or newer, other recent distros should also be fine
+    - macOS
+      - Only tested on macOS 13 (Ventura), older versions might work but untested
+  - CPU: Intel Core i5 (2nd gen) or equivalent
+    - Older might work, but untested
+  - GPU: Integrated or dedicated graphics by Intel/AMD/NVIDIA
+    - Anything > 2010 should work fine, a little older may also work
+    - Software rendering might work but untested
+  - Memory: 4 GB
+  - Disk: HDD with at least some space left for photo storage
+  - Monitor: Anything
+- Webcam
+
+```mermaid
+graph LR
+    C{Computer} -- "USB (or Integrated)" --> W(Webcam)
+```
+
+## Recommended hardware
+
+- A computer
+  - OS either:
+    - Windows 10/11
+    - Linux
+      - Ubuntu 20.04 or newer, other recent distros should also be fine
+    - macOS 13
+  - CPU: Intel Core i3 (6th gen) or equivalent
+  - GPU: Integrated or dedicated graphics by Intel/AMD/NVIDIA
+  - Memory: 8 GB
+  - Disk: SSD with enough space left for photo storage
+  - Monitor: At least 1920x1080 with touch
+- Digital camera
+  - Live view: Supported directly by gPhoto2, or using HDMI out and a USB HDMI capture device (which can act like a webcam)
+  - Capturing: Supported directly by gPhoto2
+- Optional: Photo printer
+
+```mermaid
+graph LR
+    C{Computer} -- USB --> A(Camera)
+    H["HDMI capture device (optional)"] -- USB --> C{Computer}
+    A -- HDMI --> H
+    M -- "USB (touchscreen)" --> C
+    C{Computer} -- VGA/DVI/HDMI/DP --> M(Monitor)
+    C{Computer} -- "USB/(W)LAN" --> P(Photo printer)
+
+```
+
+## Running the application
+
+```admonish warning
+Make sure to use good quality USB and HDMI cables and preferably connect any USB devices without extension cords. There have been reports of freezes in camera handling due to bad cables.
+```
 
 1. Download the latest MomentoBooth release for your OS from the [GitHub releases page](https://github.com/h3x4d3c1m4l/momento-booth/releases).
-1. Extract the zip file and run the MomentoBooth executable.
+2. Extract the zip file and run the MomentoBooth executable.
     * It may be convenient to place a shortcut on your desktop.
-1. You will be greeted by the homescreen saying "Touch to start" and a green background. Press `Ctrl+S` to open the settings panel. → [See all shortcuts](settings_general.md#hotkeys).
-1. Go to the **hardware** tab
-1. Ensure [live view method](settings_hardware.md#live-view-method) is set to "Webcam" and select your webcam from the list in the [webcam setting](settings_hardware.md#webcam). The [capture method](settings_hardware.md#capture-method) is set to "Live view source" by default.
+3. You will be greeted by the homescreen saying "Touch to start" and a green background. Press `Ctrl+S` to open the settings panel. → [See all shortcuts](settings_general.md#hotkeys).
+4. Go to the **hardware** tab
```

Comment:
Doing these numbers does not actually change anything in the output. Clarity from the code side vs having to change the numbers when you insert an additional step or remove one.

Reply:
Weird... I can't remember that I actually changed those...